tests/mtouch/MTouch.cs

```diff
@@ -3427,6 +3427,46 @@ public void NoProductAssemblyReference (bool nofastsim)
 			}
 		}
 
+		[Test]
+		[TestCase (true)]
+		[TestCase (false)]
+		public void MixedModeAssembliesCanNotBeLinked (bool nofastsim)
+		{
+			foreach (var flavor in new Tuple<MTouchLinker, bool> [] {
+					new Tuple <MTouchLinker, bool> (MTouchLinker.DontLink, false),
```

Comment:
It doesn't really matter much, but you can have more arguments to the TestCase attribute, i.e. something like this:

```csharp
[TestCase (true, MTouchLinker.DontLink, false)]
[TestCase (true, MTouchLinker.LinkAll, true)]
[TestCase (true, MTouchLinker.LinkSdk, true)]
[TestCase (false, MTouchLinker.DontLink, false)]
[TestCase (false, MTouchLinker.LinkAll, true)]
[TestCase (false, MTouchLinker.LinkSdk, true)]
public void MixedModeAssembliesCanNotBeLinked (bool nofastsim, MTouchLinker linker, bool builds_successfully)
```